Add new field to collection

JoshFen95 opened the pull request:
- Add new field generation_date to the combined_sic_activities.
- Add this new field to be populated with the current date on record creation/when create_combined_sic_activities.js is ran

JoshFen95 committed:
Add new field to collection

Add new field generation_date to the combined_sic_activities.
Add this new field to be populated with the current date on record
creation/when create_combined_sic_activities.js is ran

SteveHicksCH commented on src/scripts/create_combined_sic_activites.js:
```diff
@@ -8,13 +8,19 @@
 
 db.combined_sic_activities.drop()
-db.ons_combined_sic_activities.aggregate([{$out : "combined_sic_activities"}])
+db.ons_combined_sic_activities.aggregate([ {
```
could you do this in src/scripts/create_ons_combined_sic_activites.js so that we just have the date when this (ONS) file was generated. I am not expecting this to happen that often (maybe every 3 to 5 years)

SteveHicksCH commented on src/scripts/create_combined_sic_activites.js:
```diff
@@ -56,8 +62,13 @@ db.ch_economic_activity_sic_codes.aggregate([
-            "is_ch_activity" : 1
-        }
+            "is_ch_activity" : 1,
+            "count": { $literal : 1 }
```
Don't we want the generation_date field here

JoshFen95 committed:
Add new field to mongo data

- add new generation_date field to combined_sic_activities and
  ons_combined_sic_activities

- Update export files to contain this new field
